test/go.mod

```diff
@@ -0,0 +1,10 @@
+module gihub.exasol.com/terraform-aws-exasol/test
+
+  go 1.16
+
+  require (
+  github.com/grantstreetgroup/go-exasol-client v0.0.0-20210526002632-296e16acd559
+  github.com/gruntwork-io/terratest v0.35.4
+  github.com/sethvargo/go-password v0.2.0
+  github.com/stretchr/testify v1.7.0
+  )
```

Comment:
Is indentation off in this file?

Reply:
I think go formats all files. So probably it's intended to be like that.